src/main/java/graphql/testrunner/service/CommandExecutorService.java

```diff
@@ -0,0 +1,97 @@
+package graphql.testrunner.service;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import graphql.testrunner.util.TestRunnerException;
+
+import org.springframework.stereotype.Service;
+
+import static java.util.Arrays.asList;
+
+@Service
+public class CommandExecutorService {
+
+    public static final Logger LOGGER = Logger.getLogger(CommandExecutorService.class.getName());
+
+    public static final String GRAPHQL_DIR = "/app/graphql-java/";
+    public static final List<String> INSTALL_GIT = asList("sh", "-c", "apk add --no-cache git");
+    public static final List<String> BUILD_GRAPHQL_JMH_JAR = asList("sh", "-c", "RELEASE_VERSION=test-runner-jmh ./gradlew jmhJar");
```

Comment:
I imagine there's going to be quite a lot of commands added as this project grows and that we may (eventually) need a separate place to organize them.